apps/extension/webpack.config.js

```diff
@@ -74,12 +74,15 @@ module.exports = {
     resolve: {
         mainFields: ['browser', 'main', 'module'],
         alias: {
-            'react-native$': 'react-native-web'
+            'react-native$': 'react-native-web',
+            'shared': path.resolve(__dirname, '../../libs/shared/src'),
+            'shelter': path.resolve(__dirname, '../../libs/shelter/src'),
         },
         plugins: [new TsconfigPathsPlugin()],
         modules: [
             path.resolve(__dirname, 'node_modules'),
-            path.resolve(__dirname, '../../libs/ui/node_modules')
+            path.resolve(__dirname, '../../libs/ui/node_modules'),
+            path.resolve(__dirname, '../../libs/shelter/node_modules'),
```

Comment:
create array of external libs ['ui', 'shelter', 'shelter'] and do map for their aliases and modules